spark/interpreter/src/main/java/org/apache/zeppelin/spark/SparkVersion.java

```diff
@@ -86,7 +86,7 @@ public boolean isSpark2() {
   }
 
   public boolean isSecretSocketSupported() {
-    return this.newerThanEquals(SPARK_2_3_1);
+    return this.newerThanEquals(SPARK_2_3_1) || this.equals(SparkVersion.fromVersionString("2.1.2"));
```

Comment:
I don't think we should hardcode this? why not newerThanEquals(SPARK_2_1_2)?

Reply:
This is by accident, I have removed it.